### PR TITLE
feature: Create rating component

### PR DESCRIPTION
**Pull request by LuisDiaz95**
# Description

Creation of the rating component, to be used in the Catalog Item component to represent the rating of given item with stars

Fixes # (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested with storybook

- [ ] Test Full Stars
- [ ] Test Half Star

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/Rating/Rating.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  rating
+}) => {
+  return (
+    <div className="star-rating">
+      {[...Array(5)].map((star, index) => {
+        index += 1;
+        return (
+          <span
+            key={index}
+            className={index <= rating ? "star on" : (index-0.5) <= rating ? "star half" : "star"}
```
Comment (miguelromeroglb) [inline src/components/Rating/Rating.jsx]: Normalmente los if ternarios no son tan semánticos, siempre se recomienda usar una función que haga el "calculo"
```js
function getClassName(index, rating) {
  if(index <= rating) return "start on"
  else if(index - 0.5 <= rating) return "star half"
  return "star"
}
```

**Review by miguelromeroglb — COMMENTED** [review]
Se ve bien, faltan los tests

File: src/components/Rating/Rating.jsx (hunk 1 above)
Comment (LuisDiaz95) [reply src/components/Rating/Rating.jsx]: Corregido

**Comment by LuisDiaz95** [comment]
Tests agregados en commit 72d0c76

**Review by miguelromeroglb — APPROVED** [review]
LGTM

**Comment by LuisDiaz95** [comment]
image
image

**Pull request merged**